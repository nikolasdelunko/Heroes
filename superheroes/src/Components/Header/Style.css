.header {
  background-color: #000000bd;
  height: 73px;
  display: flex;
  align-items: center;
  justify-content: center;
}

.header-text {
  text-transform: uppercase;
  color: white;
}
