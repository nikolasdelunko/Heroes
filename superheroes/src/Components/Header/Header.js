import React from 'react'
import './Style.css'

export default function Header() {
	return (
		<div className="header">
			<h1 className="header-text">You hero list</h1>
		</div>
	)
}
