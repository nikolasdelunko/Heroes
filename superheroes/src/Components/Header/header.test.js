import React from "react";
import Header from "./Header";
import { unmountComponentAtNode } from "react-dom";
import {render, screen, waitFor} from '@testing-library/react'

let container = null;

beforeEach(() => {
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe("Test header", () => {
  test("Smoke test for header", () => {
		render(<Header />)
	});
});
