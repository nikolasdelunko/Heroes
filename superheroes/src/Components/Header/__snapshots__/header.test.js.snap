// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot Testing Header snapshot 1`] = `"<div class=\\"header\\"><h1 class=\\"header-text\\">You hero list</h1></div>"`;
